### PR TITLE
Add hard-coded images of tweets to the candidate page.

### PR DESCRIPTION
**Pull request by ganewman**
### What is a quick description of the change?

Add hard-coded images of tweets to the candidate page.

### Is this fixing an issue?

none

### Are there more details that are relevant?

none

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [  ] Tests passing
- [X] Coding style (indentation, etc)

Please explain why any are not present, if any.

### CODE REVIEW

File: project/src/main/webapp/candidate.html [hunk 1]
```diff
@@ -21,6 +21,35 @@ <h2>Voter Central Candidate Page</h2>
       <div class="candidate-page-info" id="news-articles-container">
       </div>
       <div class="candidate-page-info" id="social-media-container">
+        <div id="official-feed">
```
Comment (KathyFeiyang) [inline project/src/main/webapp/candidate.html]: nit: Perhaps rename to "candidate-feed"

**Review by KathyFeiyang — APPROVED** [review]

File: project/src/main/webapp/candidate.html [hunk 2]
```diff
@@ -21,6 +21,35 @@ <h2>Voter Central Candidate Page</h2>
...
+              working families & the movement that made it happen. Corrupt
+              interests haven’t had a good night’s sleep since Emoji:winking face
+              Thank you Emoji:purple heart"/>
+          <img src="/images/officialtweet1.png" alt="Our team has been working
+              with artists to reimagine communities into modern day, WPA-style
+              Green New Deal posters. So far we’ve got the Bronx, Queens,
+              Detroit, Boston, Los Angeles, & San Juan Emoji:Flag of Puerto Rico
+              You can find them at https://shop.ocasiocortez.com"/>
+        </div>
+        <div id="mentions-feed">
```
Comment (kellyktan) [inline project/src/main/webapp/candidate.html]: Could be nice if this was side-by-side/inline horizontally with the "Official Twitter Feed"

Comment (ganewman) [reply project/src/main/webapp/candidate.html]: I think that's the plan eventually. I will try but it will probably take a bit of work to make it happen.

**Review by tjyoung99 — APPROVED** [review]

File: project/src/main/webapp/candidate.html (hunk 2 above)
Comment (kellyktan) [reply project/src/main/webapp/candidate.html]: Kk, no need to do it now if it'll take too long (esp if you're planning to not show "mentions" until you can figure a way to moderate it somehow)

**Review by kellyktan — APPROVED** [review]

**Pull request merged**